DashAI/back/dataloaders/classes/image_dataloader.py

```diff
@@ -8,7 +8,7 @@ class ImageDataLoader(BaseDataLoader):
     """Data loader for data from image files."""
 
     def load_data(
-        self, dataset_path: str, file: UploadFile = None, url: str = None
+        self, dataset_path: str, params=None, file: UploadFile = None, url: str = None
```

Comment:
I think the signature and/or documentation of this method needs to be improved.

It is not clear to me, what is dataset_path? 
In that path will the files be stored? If so, then it should be called output_path or something like that.

Also, why specify file and url? 
If the two variables do the same thing only varying the source, then you should unify these parameters in one argument as filepath, as pandas does in its read_* methods (see read_csv).

Also, you added a `params` argument but did not specify what it does it in the documentation nor its possible type.

Reply:
change the name to output_path to make it clearer, also remove the params as it is not used in this dataloader. If about the way to upload a dataset I realized that from a url (which in written rigor is the name in the HF Hub) it is done in another way and in fact it is not necessary a specific dataloader. I suggest in that case to remove the URL parameter and then add a specific dataloader for the Hub cases.

Reply:
My mistake about the hub, the way it is used to use the urls, but in the same way to use the hub urls it would be necessary another type of dataloader.

Reply:
Yes, I think that we should implement specific dataloaders for huggingface hub datasets. 
I will appoint this in a card and accept this changes.